NativeApp/src/services/deviceStorage.js

```diff
@@ -10,6 +10,15 @@ const deviceStorage = {
     }
   },
 
+  async getItem(key) {
+    try {
+      const value = await AsyncStorage.getItem(key);
+      return value;
```

Comment:
not a big deal - but you could just do `return await AsyncStorage.getItem(key);`

Reply:
Nice spot! Prefer just returning await.